packages/db/postgres/client.js

```diff
@@ -120,7 +121,7 @@ export class PostgresClient {
    */
   async createUpload (data) {
     const now = new Date().toISOString()
-    /** @type {{ data: number, error: Error }} */
+    /** @type {{ data: number, error: PostgrestError }} */
```

Comment:
Is data a string (instead of number)?

Reply:
good catch! Probably the same error in several other client rpc calls

Reply:
Good catch, I'll update it.